src/transformers/models/sew_d/configuration_sew_d.py

```diff
@@ -72,6 +72,8 @@ class SEWDConfig(PretrainedConfig):
             The non-linear activation function (function or string) in the encoder and pooler. If string, `"gelu"`,
             `"relu"`, `"selu"`, `"gelu_python"` and `"gelu_new"` are supported.
         hidden_dropout (`float`, *optional*, defaults to 0.1):
+            Not used.
```

Comment:
If `hidden_dropout` is not used, maybe we can just remove it? cc @amyeroberts

Reply:
We'd need to do a deprecation cycle in case any users use it in their own code. So something along the lines of: 


```py
class SEWDConfig(PretrainedConfig):
    """
    ...
    hidden_dropout (`float`, *optional*, defaults to 0.1):
        Deprecated. Not used by the model and will be removed in a future version. 
    ...
    """
    def __init__(...):
        ...
        self._hidden_dropout = hidden_dropout

    @property 
    def hidden_dropout(self): 
        logger.warning_once("hidden_dropout is not used by the model and will be removed as config attribute in v4.35")
        return self._hidden_dropout

    def to_dict(self):
        """
        Serializes this instance to a Python dictionary.
        """
        output = super().to_dict()
        output["hidden_dropout"] = output.pop("_hidden_dropout")
        return output    
```

Reply:
@amyeroberts I added as you suggested.